### PR TITLE
doc center redesign

### PR DESCRIPTION
**Pull request by oleksiik-uxd**
- Issue: no hamburger menu icon for mobile version. Need dev help for that later (issue open in github)

### CODE REVIEW

File: site/docs/welcome.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
+# You don't need to edit this file, it's empty on purpose.
+# Edit theme's home layout instead if you wanna make some changes
+# See: https://jekyllrb.com/docs/themes/#overriding-theme-defaults
+
+id: welcome
+title: Hello!
```
Comment (arnonkehat) [inline site/docs/welcome.md]: This should either say "Welcome to stylable" as title, or be an empty string.

**Comment by arnonkehat** [comment]
talked to @idoros we need a mobile solution before we merge. can you tell me what's needed for that?

**Comment by oleksiik-uxd** [comment]
1. @arnonkehat write: "can you tell me what's needed for that?"
I would like to hide and open the sidemenu on mobile. 
My issue with that: it doesn't works with just css properties, we need to add some code also in `header.html` using Liquid (as I understood).
So what I need: I need a trigger button (hamburger menu, button, no matter) which can hide/open side navigation menu.

can you help me with that?

2. "Hi!" Was a mistake. I was trying to don't use the same sentence in a menu an in page. Can we use just "Welcome" ?

**Comment by arnonkehat** [comment]
Yes. Or something funny :)

**Comment by oleksiik-uxd** [comment]
Good,
so ping we when you are done with hamburger menu - and then I'll make the style changes and add a new PR.

**Pull request merged**